chia/plotting/manager.py

```diff
@@ -88,12 +145,36 @@ def save(self):
     def load(self):
         try:
             serialized = self._path.read_bytes()
+            version = uint16.from_bytes(serialized[0:2])
```

Comment:
if you're loading an old cache file, the first 2 bytes won't contain the version number, right?
So this is still a breaking change. I'm not sure it's worth the complexity of versioning this. If we fail to load it, we can just throw it out and re-create it, right?

Reply:
> if you're loading an old cache file, the first 2 bytes won't contain the version number, right?

Why shouldn't it contain the version? It's already there prior to this PR.

> So this is still a breaking change. I'm not sure it's worth the complexity of versioning this. If we fail to load it, we can just throw it out and re-create it, right?

 It's not worth the trouble for small farms because they can rebuild it fast right, but it might worth it for the large farms. However, i will have more changes to the cache soon so im going to incorporate this PR into another one i.e. im closing here.

Reply:
I see, you just can't use the version field because you can't parse before you know the version